README.md

```diff
@@ -59,24 +60,19 @@ $ cordova platform add android
 
 Set the environment variable `ANDROID_BUILD` to gradle (e.g. on mac this is `export ANDROID_BUILD=gradle`)
 
-### iOS
+#### iOS
 Add iOS as a platform
 ```console
 $ cordova platform add ios
 ```
 
-<!-- TODO update link to sample
-## Example application
-
-There is a [sample application and a quickstart guide]().
- -->
 ## Overview of the library
 
 Once the plugin has been added to a project, the basics are:
 
 Opening a Datastore
 ```js
-var DatastoreManager = require('cloudant-sync.DatastoreManager');
+var DatastoreManager = cordova.require('cloudant-sync.DatastoreManager').DatastoreManager();
```

Comment:
Opened #80 for `content_type` since being consistent with the CouchDB format seems better. So I think the doc is correct, but the code will need updating under that issue.